testgrid/tgrun/pkg/runner/vmi/vmi.go

```diff
@@ -122,8 +122,8 @@ export KURL_URL='%s'
 export KURL_FLAGS='%s'
 export KURL_UPGRADE_URL='%s'
 export SUPPORTBUNDLE_SPEC='%s'
-export OS_NAME='%s',
-export NUM_NODES='%d'
+export OS_NAME='%s'
+export NUM_NODES='%f'
```

Comment:
why decimal? not int?

shouldnt this be %d?

Reply:
%d is the format specifier for base 10 integers